<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>613</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ConfigWidget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icon3.png</normaloff>:/resources/icon3.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_del_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>721</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Set Parameters(m^2/30s)</string>
   </property>
   <widget class="QPlainTextEdit" name="Qw_pos">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>40</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e4</string>
    </property>
   </widget>
   <widget class="QLabel" name="Qw_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>45</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer noise (Qw):</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Qw_zwd">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>40</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>3e-8</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pk_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial covariance (Pk):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>dynamic_pos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>zwd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>clk</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Qw_clk">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>40</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>568</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>amb</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Qw_amb">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>40</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e-16</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>658</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>ion</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Qw_ion">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>40</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pk_amb">
    <property name="geometry">
     <rect>
      <x>539</x>
      <y>80</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e6</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pk_pos">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>80</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e3</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pk_ion">
    <property name="geometry">
     <rect>
      <x>632</x>
      <y>80</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pk_clk">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>80</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e6</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Pk_zwd">
    <property name="geometry">
     <rect>
      <x>352</x>
      <y>80</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>123</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrier/Pseudorange precision (m):G</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="L_precision">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>0.03</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="P_precision">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>3.0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="P_precision_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>3.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrier/Pseudorange precision (m):C2</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="L_precision_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>0.03</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="P_precision_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>3.0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="L_precision_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>0.03</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Carrier/Pseudorange precision (m):E</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="P_precision_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>3.0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="L_precision_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>0.03</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="P_precision_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>3.0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="L_precision_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>0.03</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>C3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>281</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Helmert variance covariance estimation</string>
   </property>
   <widget class="QCheckBox" name="Helmert_VC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Helmert VC select</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>721</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>stochastic model</string>
   </property>
   <widget class="QComboBox" name="stochastic_model">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Weighted sine elevation angle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Weighted cosine elevation angle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sine  elevation angle 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Panda stochastic model</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>341</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Ratio Test</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ratio value:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Ratio_value">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>104</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>2.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>WL/EWL fixed ambiguity weight</string>
   </property>
   <widget class="QPlainTextEdit" name="WL_weight">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>1e6</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>NL ambiguity cutoff</string>
   </property>
   <widget class="QPlainTextEdit" name="NL_cutoff">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>85</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="plainText">
     <string>0.5</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
